dist/src/main/resources/wildfly/patches/standalone.xsl

```diff
@@ -349,7 +354,7 @@
                  numAvailSchedulerThreads="3">
         <xsl:attribute name="enabled">
           <xsl:choose>
-            <xsl:when test="$kettle.build.type='dev'">true</xsl:when>
+            <xsl:when test="$kettle.build.type='dev'">${hawkular.agent.enabled:true}</xsl:when>
```

Comment:
Escaping of dollar signs in XSL is tricky. I have run the build like this and it produces what it was supposed to produce. Let's merge like this and revisit later if necessary.